values.yaml

```diff
@@ -53,6 +53,17 @@ enterprise:
       #  roles:
       #    - "admin"
 
+##
+## Service Account
+##
+serviceAccount:
+  # Whether the Chart should create the service account or not
+  create: true
+  # Used to define service account annotations
+  serviceAccountAnnotations: {}
+  # Used to override service account name
+  serviceAccountName:
```

Comment:
Why not put this per service rather than a centrally located serviceAccount?

For example, what if we wanted st2stream to use a different serviceaccount from st2api?

Reply:
Following the other official Helm charts and templating examples from the https://github.com/helm/charts/search?q=serviceaccount&unscoped_q=serviceaccount looks like it's pretty good default to start with. There is however a good opportunity to extend the base functionality in future, if there will be advanced use cases you mentioned.

Reply:
Hello,
sorry for the delay.
I added a new version in the CHANGELOG, is this what I was supposed to do ?

Reply:
Hello @mickmcgrath13,
I asked myself the question but it felt like a good start.
Personally I only need a service account and I did not have a use case in mind for having multiple service accounts.

But we can easily iterate over this first step.